package.json

```diff
@@ -46,6 +46,11 @@
     "docker:build:artifacts": "docker rmi --force api3/airnode-artifacts && docker build --tag api3/airnode-artifacts:latest --file docker/Dockerfile .",
     "docker:build:deployer": "docker build --tag api3/airnode-deployer:latest --file packages/airnode-deployer/docker/Dockerfile .",
     "docker:build:node": "docker build --tag api3/airnode-client:latest --file packages/airnode-node/docker/Dockerfile .",
+    "docker:buildx": "docker run --privileged --rm tonistiigi/binfmt --install all && yarn docker:buildx:admin && docker:buildx:artifacts && yarn docker:buildx:deployer && yarn docker:buildx:node",
```

Comment:
These commands use BuildKit/buildx to build images for various architectures (amd64 and arm64 here) on the same host machine (which can be either architecture). The commands also pushe the images to Docker Hub as multi-arch compatiblle. Probably inappropriate for inclusion here, what do you think?

Reply:
I will remove these in favour of Confluence documentation.